src/common/locations.ts

```diff
@@ -1,8 +1,30 @@
 /** Helpers for dealing with the State / Counties dataset. */
 import US_STATE_DATASET from 'components/MapSelectors/datasets/us_states_dataset_01_02_2020.json';
+import { each, uniqBy } from 'lodash';
 
 // TODO(michael): Add types, move more common code here.
 
+export function getLocationNames() {
+  const locations = US_STATE_DATASET.state_dataset.map(state => {
+    return {
+      ...state,
+      full_fips_code: state.state_fips_code,
+    };
+  });
+
+  each(US_STATE_DATASET.state_county_map_dataset, (value, key) => {
+    locations.push(
+      ...value.county_dataset.map(county => {
+        return { ...county, state: county.state_code };
+      }),
+    );
+  });
+
+  // Currently check for the unique fips so there aren't duplicates, but maybe
+  // match the way that the homepage gets unique values
```

Comment:
Isn't this going to be kinda' arbitrary?

E.g. now "Hancock County, IN" isn't in the list because instead there's a weird "Hancock County / Marion County, IN" entry (which I guess is using the Hancock County FIPS)...

I think we either need to:
1) Actually use the same logic as the homepage, or
2) Just delete these "Foo County / Bar County" entries from the dataset so we don't need special handling.  I think these existed in order to allow you to search by city (for cities that span multiple counties), but we don't really support that anymore.

Reply:
okay yes removed the slash in code to prevent this